lib/tests/streamlit/session_state_test.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2018-2021 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest.mock import MagicMock, patch
+
+from streamlit.errors import StreamlitAPIException
+from streamlit.report_thread import _StringSet
+from streamlit.session_state import SessionState
+
+
+# TODO: Mock widget values and modify the tests below to account for this once
+#       session_state and widget state are unified.
+class SessionStateTests(unittest.TestCase):
```

Comment:
I wonder if the tests would be less noisy/verbose if we took advantage of pytest's assert rewriting

Reply:
Plain asserts definitely look much nicer here -- switched to using them over `unittest` style asserts